runtime/service/src/main/java/org/apache/polaris/service/auth/internal/service/OAuthError.java

```diff
@@ -16,21 +16,25 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.polaris.service.auth;
 
-import jakarta.annotation.Nullable;
+package org.apache.polaris.service.auth.internal.service;
 
-/**
- * This is a union of configuration settings of all token brokers.
- *
- * @see TokenBrokerFactory
- */
-public interface TokenBrokerFactoryConfig {
-  @Nullable
-  String file();
+public enum OAuthError {
+  invalid_request("The request is invalid"),
```

Comment:
nit: UPPER case or CamelCase for values? (usage sites have changes anyway).

Reply:
In fact, these are error codes defined in the RFC: https://datatracker.ietf.org/doc/html/rfc6749#section-5.2
Since the enum constants are included as-is in the response, I believe we should not modify them.

Reply:
Good point!